schemas/order.json

```diff
@@ -154,6 +154,23 @@
         }
       ]
     },
+    "booking_channel": {
+      "required": false,
+      "type": "array",
+      "description": "Reseller or other type of api client that the order was created by. This is available only for particular API clients.",
+      "items": [
+        {
+          "type": "object",
+          "properties": {
+            "name": {
+              "required": true,
+              "type": "sting",
```

Comment:
@ap-bookingkit I think it's supposed to be "string" :smile:

Reply:
@darkbind Yeah, you are right, updated all such instances.